networkx/algorithms/bipartite/basic.py

```diff
@@ -126,10 +127,21 @@ def is_bipartite_node_set(G, nodes):
 
     Notes
     -----
+    An exception is raised if the input nodes are not distinct, because in this
+    case some bipartite algorithms will yield incorrect results.
```

Comment:
Instead of putting this in the Notes, it would be better to have this in a `Raises` section, something like:

```rst
Raises
------
AmbiguousSolution : 
    If `nodes` contains duplicates.
```